LayoutKit/Layouts/TextViewLayout.swift

```diff
@@ -0,0 +1,186 @@
+                  textContainerInset: textContainerInset,
+                  layoutAlignment: layoutAlignment,
+                  flexibility: flexibility,
+                  viewReuseId: viewReuseId,
+                  config: config)
+    }
+
+    public convenience init(attributedText: NSAttributedString,
+                            font: UIFont? = nil,
+                            lineFragmentPadding: CGFloat = 0,
+                            textContainerInset: UIEdgeInsets = .zero,
+                            layoutAlignment: Alignment = defaultAlignment,
+                            flexibility: Flexibility = defaultFlexibility,
+                            viewReuseId: String? = nil,
+                            config: ((TextView) -> Void)? = nil) {
+        self.init(text: .attributed(attributedText),
+                  font: font,
+                  lineFragmentPadding: lineFragmentPadding,
+                  textContainerInset: textContainerInset,
+                  layoutAlignment: layoutAlignment,
+                  flexibility: flexibility,
+                  viewReuseId: viewReuseId,
+                  config: config)
+    }
+
+    // MARK: - Layout protocol
+
+    open func measurement(within maxSize: CGSize) -> LayoutMeasurement {
+        let fittedSize = textSize(within: maxSize)
+        let decreasedToSize = fittedSize.decreasedToSize(maxSize)
+        return LayoutMeasurement(layout: self, size: decreasedToSize, maxSize: maxSize, sublayouts: [])
+    }
+
+    open func arrangement(within rect: CGRect, measurement: LayoutMeasurement) -> LayoutArrangement {
+        let frame = alignment.position(size: measurement.size, in: rect)
+        return LayoutArrangement(layout: self, frame: frame, sublayouts: [])
+    }
+
+    // MARK: - private helpers
+
+    private func textSize(within maxSize: CGSize) -> CGSize {
+        var insetMaxSize = maxSize.decreased(by: textContainerInset)
+        insetMaxSize.width -= lineFragmentPadding * 2
+
+        let size = self.isTextEmpty
+            ? text.textSizeWithEmptyText(within: insetMaxSize, font: font ?? TextViewLayout.defaultFont(withText: text))
+            : text.textSize(within: insetMaxSize, font: font ?? TextViewLayout.defaultFont(withText: text))
+
+        var textSize = size.increased(by: textContainerInset)
+        textSize.width += lineFragmentPadding * 2
+        return textSize
+    }
+
+    private static func defaultFont(withText text: Text) -> UIFont {
+        switch text {
+        case .unattributed(_):
+            return TextViewDefaultFont.unattributedTextFont
+        case .attributed(let attributedText):
+            return attributedText.length == 0
+                ? TextViewDefaultFont.attributedTextFontWithEmptyString
+                : TextViewDefaultFont.attributedTextFont
+        }
+    }
+
+    // MARK: - overriden methods
+
+    open override func configure(view textView: TextView) {
+        config?(textView)
```

Comment:
Instead of this call, better to call super.configure in case that gets some other valuable code added to it at some point.